pipeline/src/transformers/eventDocument.ts

```diff
@@ -0,0 +1,131 @@
+import { PrismicDocument, TimestampField, asDate } from "@prismicio/client";
+import {
+  EventPrismicDocument,
+  WithEventFormat,
+  WithLocations,
+  WithInterpretations,
+} from "../types/prismic/eventDocuments";
+import {
+  EventDocumentFormat,
+  EventDocumentLocation,
+  EventDocumentInterpretation,
+} from "../types/transformed/eventDocument";
+import { ElasticsearchEventDocument } from "../types/transformed";
+import {
+  isFilledLinkToDocumentWithData,
+  isImageLink,
+  asText,
+  asTitle,
+  isNotUndefined,
+} from "../helpers/type-guards";
+import { linkedDocumentIdentifiers, formatSeriesForQuery } from "./utils";
+
+const onlineLocation = {
+  type: "EventLocation",
+  id: "ef04c8e3-26be-4fbc-9bef-f52589ebc56c",
```

Comment:
Where is this id from? (not Prismic?)

Reply:
As discussed, isOnline is a boolean in Prismic. We want to treat it as a location so it needs to be in the same format as physical location